pkg/ready/ready.go

```diff
@@ -47,12 +48,18 @@ type fileReady struct{}
 func (r fileReady) Set() error {
 	f, err := os.Create(FileName)
 	if err != nil {
+		if os.IsExist(err) {
```

Comment:
I played around with `os.Create()` and it actually doesn't return an error when the file exists already (just returns the file). In spite of that knowledge I think it's better to handle the cases where `err != nil` completely with the additional check to see if it's an error because it exists.

Reply:
`os.Create` will truncate the file if it exists though. Will that be a problem? If not then okay.

Reply:
I'm not an expert...so I just created a program that called `Set()` a million times without issue. I don't believe it will be a problem.

Reply:
As long as the file you do not care about the contents of an existing file before running `os.Create` you will be okay. I just wanted to make sure we weren't overwriting file contents we cared about (or the customer cared about).

Reply:
We may want to `os.Stat()` the file before calling create, as we do in `Unset()`.

Reply:
Overwriting the file on every `Set()` is acceptable. I don't intend to make the change unless someone specifically requests it on this PR.

Reply:
For this change, I'm okay with just using `Create`.